### PR TITLE
Filter out non-DX types

### LINKED ISSUE #150: types endpoint failing with AttributeError: lookupSchema
```
GET http://localhost:8080/Plone/@types/Document
```

```
2016-10-22 12:06:19 ERROR Zope.SiteErrorLog 1477152379.920.0579750084626 http://localhost:8080/Plone/@types/Document
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module plone.rest.service, line 20, in __call__
  Module plone.restapi.services, line 15, in render
  Module plone.restapi.services.types.get, line 50, in reply
  Module plone.restapi.types.utils, line 134, in get_jsonschema_for_portal_type
  Module plone.restapi.types.utils, line 108, in get_jsonschema_for_fti
  Module plone.restapi.types.utils, line 37, in get_ordered_fields
AttributeError: lookupSchema
```

(On both Plone 4.3.x and Plone 5.0.4)

### PR DESCRIPTION
**Pull request by jaroel**
Fixes #150

### CODE REVIEW

**Comment by jaroel** [comment]
I know the tests are failing for this, but let me think what you think of this approach first.
I'm doubting between this and just returning a bare bones dict in the actual types call. So `@types/Plone Site` would return a dict without fields etc.

**Comment by lukasgraf** [comment]
@jaroel I'd be ok with this approach, but also the barebones dict one - I don't have a strong opinion either way. Eventually, we might possibly want to extend this to support Archetypes FTIs (no priority for me though), but I don't think either approach limits us in adding that later.

File: CHANGES.rst [hunk 1]
```diff
@@ -7,6 +7,11 @@ Changelog
 - tweaks to README.rst
   [tkimnguyen]
 
+Bugfixes:
+
+- Don't list non-DX types. Refs https://github.com/plone/plone.restapi/issues/150
```
Comment (tisto) [inline CHANGES.rst]: @jaroel we should mention the types endpoint here. :)

**Comment by tisto** [comment]
LGTM. Just slightly amend the changelog and make sure the tests pass.

**Pull request merged**